### ISSUE TITLE
I can't read a file in polish special character folder name

### ISSUE THREAD

**Issue by queleanalytics** [tidyverse/readxl#624]
I' am trying to export folder names and read .xlsx files under the folder.
When i come across to polish folders due to special characters R cannot read it.
Any help?
Here is the example.
I can read the file in a normal folder name.

```
> my_data1 <- read_excel("C:/data/Projects/Credit File conversion/Poland/Data_test/Credit upload file_Q42019 .xlsx")
New names:
* `` -> ...2
* `` -> ...3
* `` -> ...4
* `` -> ...5
* `` -> ...6
* ... and 1 more problem
> 
> 
And when i try to read it under the folder SUWAŁA.
> Sys.setlocale("LC_ALL", "Polish")

> my_data2 <- read_excel("C:/data/Projects/Credit File conversion/Poland/Data_test/SUWAŁA/Credit upload file_Q42019 .xlsx")

Error: Evaluation error: zip file 'C:\data\Projects\Credit File conversion\Poland\Data_test\SUWAŁA\Credit upload file_Q42019 .xlsx' cannot be opened.
```

**Comment by queleanalytics** [comment]
Here attached is the file to be put in a folder name :   SUWAŁA
[Credit upload file_Q42019 .xlsx](https://github.com/tidyverse/readxl/files/4852385/Credit.upload.file_Q42019.xlsx)

**Comment by queleanalytics** [comment]
Note the polish alphabetic :  SUWAŁA

**Comment by jennybc** [comment]
I can replicate this problem, even after I install Polish language support in my Windows VM. I'm afraid that the `"Ł"` poses special challenges for R, since it cannot be represented in `"latin1"`. I think our best hope is that the soon-to-be-released R 4.2 will save us from these headaches, with it support for UTF-8 as native encoding on Windows.

https://developer.r-project.org/Blog/public/2021/12/07/upcoming-changes-in-r-4.2-on-windows/index.html
